content/administration/maintain/azure_oauth.rst

```diff
@@ -188,24 +188,102 @@ configuration automatically loads the :guilabel:`token` in Odoo, and a tag stati
 Finally, click :guilabel:`Test Connection`. A confirmation message should appear. The Odoo database
 can now send safe, secure emails through Microsoft Outlook using OAuth authentication.
 
+.. _azure_oauth/notifications:
+
+Configuration with a single outgoing mail server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configuring a single outgoing server is the simplest configuration available for Microsoft Azure
+and it doesn't require extensive access rights for the users in the database.
+
+A generic email address would be used to send emails for all users within the database. For example
+it could be structured with a `notifications` alias (`notifications@example.com`) or `info` alias
+(info@example.com). This address must be set as the :guilabel:`FROM Filtering` on the server. This
+address must also match the `{mail.default.from}@{mail.catchall.domain}` key combination in the
+system parameters.
+
+.. seealso::
+   For more information on the from filter visit :ref:`this documentation
+   <email_communication/default>`.
+
+.. note::
+   The :guilabel:`System Parameters` can be accessed by activating
+   :doc:`../../applications/general/developer_mode` in the :menuselection:`Settings --> Technical
+   --> Parameters --> System Parameters` menu.
+
+When using this configuration, every email that is sent from the database will use the address of
+the configured `notification` mailbox. However it should be noted that the name of the sender will
+appear but their email address will change:
+
+.. image:: azure_oauth/from-name-remain.png
+   :align: center
+   :alt: Name from real sender with static email.
+
+.. example::
+   Single outgoing mail server configuration:
+
+   - Outgoing mail server login = `notifications@example.com`
+   - Outgoing mail server From filter = `notifications@example.com`
+   - `mail.catchall.domain` in system parameters = `example.com`
+   - `mail.default.from` in system parameters = `notifications`
+
 Multiple user configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Each user should have a separate server set up. The :guilabel:`from-filter` should be set so that
-only the user's email is sent from that server. In other words, only a user with an email address
-that matches the set :guilabel:`from-filter` is able to use this server.
+In addition to a generic email server, individual email servers can be set up for users in a
+database. These email addresses must be set as the :guilabel:`FROM Filtering` on each individual
+server for this configuration to work.
+
+This configuration is the more difficult of the two Microsoft Azure configurations, in that it
+requires all users configured with email servers to have access rights to settings to establish a
+connection to the email server.
```

Comment:
I have clarified this sentence a bit. I am leaving the work-around out for now, since this isn't standard anymore. 

Let me know what you think. 

Thanks,
Tim